attestationreport/bioseventdata.go

```diff
@@ -652,7 +651,7 @@ func parseEFISignaturedb(buf *bytes.Buffer, signatureDBSize int) []SignatureData
 				hash.Hash = parseVariableDataHash_GUID(buf, SHA384_DIGEST_LEN)
 				hashes = append(hashes, hash)
 			default:
-				logrus.Print("Signature GUID " + sigdb.SignatureTypeGUID)
+				log.Print("Signature GUID " + sigdb.SignatureTypeGUID)
```

Comment:
log.Tracef("Signature GUID: %v", sigdb.SignatureTypeGUID)

Do not use log.Print[f/ln]